Template/Tag/EtrackerTag.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\TagManager\Template\Tag;
+
+use Piwik\Settings\FieldConfig;
+use Piwik\Validators\NotEmpty;
+
+class EtrackerTag extends BaseTag
+{
+	const PARAM_ETRACKER_CONFIG = 'etrackerConfig';
+	
+	public function getIcon()
+	{
+		return 'plugins/TagManager/images/icons/etracker.svg';

```

Comment:
do you mind replacing tabs with 4 spaces?

Reply:
Done